ci-bench/src/cli.rs

```diff
@@ -0,0 +1,38 @@
+use clap::{Parser, Subcommand, ValueEnum};
+use std::path::PathBuf;
+
+#[derive(Parser)]
+#[command(about)]
+pub struct Cli {
+    #[command(subcommand)]
+    pub command: Command,
+}
+
+#[derive(Subcommand)]
+pub enum Command {
+    /// Run all benchmarks and prints the measured CPU instruction counts in CSV format
+    RunAll,
+    /// Run a single benchmark at the provided index (used by the bench runner to start each benchmark in its own process)
+    RunSingle { index: u32, side: Side },
+    /// Compare the results from two previous benchmark runs and prints a user-friendly markdown overview
+    Compare {
+        baseline_input: PathBuf,
+        candidate_input: PathBuf,
+    },
+}
+
+#[derive(Copy, Clone, ValueEnum)]
+pub enum Side {
```

Comment:
Maybe we can use `rustls::Side` instead?

Reply:
Unfortunately we need our own type, so we can derive `ValueEnum` which is required by `clap`

Reply:
I guess. Alternatively we could have rustls implement `FromStr` for `Side`, but that might not be much better.

Reply:
I can't imagine that `FromStr` implementation would be useful anywhere else, so in my eyes the current solution is preferrable